src/services/ProcessorService.js

```diff
@@ -489,8 +489,8 @@ processCreate.schema = {
     payload: joi.object().keys({
       resource: joi.string().required(),
       id: joi.numberId(),
-      name: joi.string().max(200).required(),
-      description: joi.string().max(10000).allow('').allow(null).required(),
+      name: joi.string().required(),
+      description: joi.string().allow(null).allow('').optional(),
```

Comment:
Should we keep it (description) required? I mean, though we are accepting `null` and empty strings, we still want to make sure the caller is intentionally passing those values i.e. `description` is not `undefined`.

Reply:
We don't require the `description` to be passed in `Project Service` https://github.com/topcoder-platform/tc-project-service/blob/develop/src/routes/projects/create.js#L30 during project creation. Do we have any particular reason for requiring it here?

Reply:
Makes sense.